src/components/ActivityList/index.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState } from 'react';
+import { BarChart, Bar, XAxis, YAxis, Tooltip, ResponsiveContainer, CartesianGrid } from 'recharts';
+import activities from '@/static/activities.json';
+import styles from './style.module.css';
+
+const ActivityCard = ({ period, summary, dailyDistances, interval, activityType }) => {
+    const generateLabels = () => {
+        if (interval === 'month') {
+            const [year, month] = period.split('-').map(Number);
+            const daysInMonth = new Date(year, month, 0).getDate(); // 获取该月的天数
+            return Array.from({ length: daysInMonth }, (_, i) => i + 1);
+        } else if (interval === 'week') {
+            return Array.from({ length: 7 }, (_, i) => i + 1);
+        } else if (interval === 'year') {
+            return Array.from({ length: 12 }, (_, i) => i + 1); // 生成1到12的月份
+        }
+        return [];
+    };
+
+    const data = generateLabels().map((day) => ({
+        day,
+        距离: (dailyDistances[day - 1] || 0).toFixed(2), // 保留两位小数
+    }));
+
+    const formatTime = (seconds) => {
+        const h = Math.floor(seconds / 3600);
+        const m = Math.floor((seconds % 3600) / 60);
+        const s = seconds % 60;
+        return `${h}h ${m}m ${s}s`;
+    };
+
+    const formatPace = (speed) => {
+        if (speed === 0) return '0:00';
+        const pace = 60 / speed; // min/km
+        const minutes = Math.floor(pace);
+        const seconds = Math.round((pace - minutes) * 60);
+        return `${minutes}:${seconds < 10 ? '0' : ''}${seconds} min/km`;
+    };
+
+    // 计算 Y 轴的最大值和刻度
+    const yAxisMax = Math.ceil(Math.max(...data.map(d => parseFloat(d.距离))) + 10); // 取整并增加缓冲
+    const yAxisTicks = Array.from({ length: Math.ceil(yAxisMax / 5) + 1 }, (_, i) => i * 5); // 生成等差数列
+
+    return (
+        <div className={styles.activityCard}>
+            <h2 className={styles.activityName}>{period}</h2>
+            <div className={styles.activityDetails}>
+                <p><strong>总距离:</strong> {summary.totalDistance.toFixed(2)} km</p>
```

Comment:
We have a lot of international users, can you translate the text into English according to the [config](https://github.com/yihong0618/running_page/blob/fb98ab9c4edd69726a9bde34ffce458c66b7378e/src/utils/const.ts#L51)?